data_hub_api/docmaps/v2/codecs/evaluation.py

```diff
@@ -158,16 +161,54 @@ def get_docmap_evaluation_type_form_tags(
     return None
 
 
+def get_docmap_actor_for_review_article_type() -> DocmapActorAnonymous:
+    return {
+        'name': 'anonymous',
+        'type': 'person'
+    }
+
+
 def get_docmap_evaluation_participants_for_review_article_type() -> Sequence[DocmapParticipant]:
     return [{
-        'actor': {
-            'name': 'anonymous',
-            'type': 'person'
-        },
+        'actor': get_docmap_actor_for_review_article_type(),
         'role': 'peer-reviewer'
     }]
 
 
+def get_docmap_affiliation(
+    editor_detail: ApiEditorDetailInput
+) -> DocmapAffiliation:
+    if editor_detail['country']:
+        return {
+            'type': 'organization',
+            'name': editor_detail['institution'],
+            'location': (
```

Comment:
Could extract this as a function, then you may not need the if statement here that otherwise returns the same?